core/ipc/sink/camel/server/src/main/java/org/opennms/core/ipc/sink/camel/server/CamelMessageConsumerManager.java

```diff
@@ -69,6 +72,8 @@
     @Autowired
     private Identity identity;
 
+    private JmxReporter jmxReporter = null;
```

Comment:
Do we need to do something similar for Kafka IPC?

Reply:
KafkaMessageConsumer seems to have only one instance of JmxReporter